sections/03_categorize.md

```diff
@@ -290,80 +290,95 @@ by allowing more efficient use of the costly practice of manual annotation.
+facilities have to merge. As a result, EHR can be less complete and less
+objective than expected.
+
+In the wider picture, standards for EHRs are many and evolving. Proprietary
+systems, indifferent and scattered use of health information standards and
+controlled terminologies makes combining and comparison of data across systems
+challenging [@url:http://www.ncrr.nih.gov/
+publications/informatics/ehr.pdf][@doi:10.1016/j.jbi.2014.10.006]. Further
+diversity arises from variation in languages, healthcare practices and
+demographics. Merging EHR gathered in different systems (and even under
+different assumptions) is challenging [@isbn:9783319448398].
+
+Combining or replicating studies across systems thus requires controlling for
+both the above biases and dealing with mismatching standards. This has the
+practical effect of reducing cohort size, limiting statistical significance,
+preventing the detection of weak effects
+[@doi:10.1590/2176-9451.19.4.027-029.ebo] and restricting the number of
+parameters that can be trained in a model. Further, rules-based algorithms have
+been popular in EHR-based research but because these are developed at a single
+institution and trained with a specific patient population they do not transfer
+easily to other populations [@doi:10.1136/amiajnl-2013-001935]. For example,
+Wiley et al. [@doi:10.1142/9789813207813_0050] showed that warfarin dosing
+algorithms often under-perform in African Americans, illustrating that some of
+these issues are unresolved even at a treatment best practices level. Lack of
+standardization also makes it challenging for investigators skilled in deep
+learning to enter the field, as numerous data processing steps must be performed
+before algorithms are applied.
+
+Finally, even if data were perfectly consistent and compatible across systems, attempts to share and combine EHR data face considerable legal and ethical barriers. Patient privacy can severely restrict the sharing and use of EHR [@doi:https://doi.org/10.1093/ije/dyn022]. Here again, standards are heterogeneous and evolving but often EHR data can often not be exported or even accessed directly for research purposes without appropriate consent. Once again, this has the effect of making data gathering more laborious, expansive and reducing sample size and study power.
+
+Several technological solutions have been proposed in this area. Solutions like
+DataShield [@doi:https://doi.org/10.1093/ije/dyu188] and ViPAR
+[@doi:10.1093/ije/dyv193] allows querying and combining of datasets and the
+calculation of summary statistics across sites without compromising patient
+identity. Continuous Analysis [@doi:10.1038/nbt.3780] can allow reproducible
+computing on private data. Using such techniques intermediate results can be
+automatically tracked and shared without sharing the original data.  which may
+help to address concerns around physician confidence. The EH4CR project
```

Comment:
Can you elaborate on this to provide a more practical framing? Is EH4CR fast enough to use in model training, for instance? Just wondering if we can bring this back to the person jumping into the field.